### PR TITLE
Buff Medical due to Shitmed Changes

### PR DESCRIPTION
**Pull request by Tmanzxd**
## About the PR

+ Buffed applicable medication heal values.
+ Buffed applicable medication stack sizes.
+ Slightly buffed dylovene, burn, and brute chems.
+ Buffed applicable medication doAfter delay from 3s to 2s.

## Why / Balance

This change was made with the advent of the new Limb Damage system. This PR seeks to achieve making SS14 Medical more equivalent to SS13 Medical when it comes to healing damage values.

This was done as Bruise Packs and Ointment only heal 5 Brute or 5 Burn respectively when applied to a limb. When each limb has a 100 Crit threshold, this can make it extremely hard even with surgery to heal limbs in the absence of Chemicals.

Furthermore on higher pop, this will lead to people burning through applicable meds even faster than before Limb Damage was implemented.

## Technical details

Mostly YML Value Changes and 2 float val changes for DoAfters. Commented old value and reason for change on each.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Should values be overtuned, they can be reduced in a future PR or before this PR is merged.

**Changelog**

:cl:
- tweak: Increased applicable medication heal values.
- tweak: Increased stack size of applicable medications.
- tweak: Slightly increased dylovene, burn, and brute chemicals heal values.
- tweak: Decreased Medical item application time from 3s to 2s

## Release Notes

- **New Features**
	- Enhanced healing items with increased effectiveness and new entities introduced.
  
- **Adjustments**
	- Healing item damage values have been significantly increased for various items, improving their in-game utility.
	- Stack counts for several healing items have been increased from 10 to 15, and prices have been adjusted accordingly.
  
- **Rebalancing**
	- Reagents have undergone a comprehensive rebalancing, with many damage values increased to enhance their impact in gameplay.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 1]
```diff
@@ -123,16 +123,16 @@
       - Biological
     damage:
       groups:
-        Brute: -15 # 5 for each type in the group
+        Brute: -45 # was 5 for each, Buffed due to limb damage changes (Goobstation)
```
[the remark this answers is not in the document]
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:126]: Its a 3x buff stupid machine.

**Comment by gluesniffler** [comment]
based and shitmedpilled

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tmanzxd) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:126]: Understandable have a nice day

**Comment by JoeHammad1844** [comment]
we're back

**Pull request merged**